packages/react/src/components/RadioButton/RadioButton.stories.js

```diff
@@ -65,7 +65,7 @@ export const Skeleton = () => {
 
 export const Playground = (args) => {
   return (
-    <RadioButtonGroup labelText="Radio Button group" {...args}>
+    <RadioButtonGroup name="Radio Button group" {...args}>
```

Comment:
Can we use `legendtText` so we have a visible label?

```suggestion
    <RadioButtonGroup legendText="Radio Button group" {...args}>
```

Reply:
Yep, `name` is still needed as it is a required property though.